content/en/docs/contributing/release-process.md

```diff
@@ -111,30 +154,66 @@ The process for cutting a minor release is as follows:
      the GitHub project to be able to push to the release branch.
 
      ```bash
+     # Must be run from the cert-manger repo folder.
      git push --set-upstream origin release-1.0
      ```
 
 4. Generate and edit the release notes:
 
-   1. Run as follows, substituting the current and last versions where appropriate:
+   1. Use the following two tables to understand how to fill in the 4
+      environment variables needed for the next step:
+
+      | Variable          | Description                           |
+      | ----------------- | ------------------------------------- |
+      | `START_REV`\*     | The git tag of the "previous"\* release |
+      | `END_REV`         | Name of your release branch           |
+      | `BRANCH`          | Name of your release branch           |
+      | `RELEASE_VERSION` | The git tag without the leading `v`   |
+
+      Examples for each release type (e.g., initial alpha release):
+
+      | Variable          | Example 1             | Example 2                | Example 3     | Example 4     |
+      | ----------------- | --------------------- | ------------------------ | ------------- | ------------- |
+      |                   |                       |                          |               |               |
+      |                   | initial alpha release | subsequent alpha release | final release | patch release |
+      |                   | `v1.3.0-alpha.0`      | `v1.3.0-alpha.1`         | `v1.3.0`      | `v1.3.1`      |
+      |                   |                       |                          |               |               |
+      | `START_REV`\*     | `v1.2.0`              | `v1.3.0-alpha.0`         | `v1.2.0`      | `v1.3.0`      |
+      | `END_REV`         | `release-1.3`         | `release-1.3`            | `release-1.3` | `release-1.3` |
+      | `BRANCH`          | `release-1.3`         | `release-1.3`            | `release-1.3` | `release-1.3` |
+      | `RELEASE_VERSION` | `1.3.0-alpha.0`       | `1.3.0-alpha.1`          | `1.3.0`       | `1.3.1`       |
+
+      > \*The git tag of the "previous" release (`START_REV`) depends on which
+      > type of release you count on doing. Look at the above examples to
+      > understand a bit more what those are.
+
+      After finding out the value for each of the 4 environment variables, set
+      the variables in your shell (for example, following the example 1):
+
+        ```sh
+        export START_REV="v1.2.0"
+        export END_REV="release-1.3"
+        export BRANCH="release-1.3"
+        export RELEASE_VERSION="1.3.0-alpha.0"
+        ```
 
-        ```bash
+   2. Generate the `release-notes.md` at the root of your the cert-manager repo
```

Comment:
> Generate `release-notes.md` at the root of your cert-manager repo

(remove both "the"s)

Reply:
Thanks! Fixed in 2909939 👍

Reply:
Done in 8f9273fd6c45b3819f7563cf7245da5e02a504ae! thanks!

Reply:
Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba